crates/re_arrow_store/src/store_view.rs

```diff
@@ -0,0 +1,254 @@
+use parking_lot::RwLock;
+
+use crate::{DataStore, StoreEvent};
+
+// ---
+
+// TODO(cmc): Not sure why I need the extra Box here, RwLock should be `?Sized`.
+type SharedStoreView = RwLock<Box<dyn StoreView>>;
+
+/// A [`StoreView`] subscribes to atomic changes in one or more [`DataStore`]s through [`StoreEvent`]s.
```

Comment:
Right now one always subscribes to all `DataStore`, whether that's a good idea in the long run or not I don't know yet, but it seems the comment is incorrect as-is?

Reply:
> super-nit: This is your field not mine, but I thought the more technical correct term is `transactional changes` instead of `atomic`? Since the changes happen in one a single transaction but not necessarily atomic in the sense of an atomic operation

Atomicity is a property of the query model, transactions are just one way of achieving it (case in point: we don't have transactions!).

Reply:
> Right now one always subscribes to all `DataStore`, whether that's a good idea in the long run or not I don't know yet, but it seems the comment is incorrect as-is?

That's pretty much what I meant, I can replace with "all".